engine/class_modules/sc_demon_hunter.cpp

```diff
@@ -5945,6 +5945,22 @@ std::unique_ptr<expr_t> demon_hunter_t::create_expression( util::string_view nam
       return this->get_target_data(*primary_idx)->dots.fiery_brand->is_ticking();
     });
   }
+  else if ( util::str_compare_ci( name_str, "seething_fury_threshold" ) )
+  {
+    return expr_t::create_constant( "seething_fury_threshold",
+                                    this->set_bonuses.t30_havoc_2pc->effectN( 1 ).base_value() );
+  } 
+  else if ( util::str_compare_ci( name_str, "seething_fury_spent" ) )
+  {
+    return make_fn_expr( "seething_fury_spent",
+                         [ this ] { return this->set_bonuses.t30_havoc_2pc_fury_tracker; } );
+  }
+  else if ( util::str_compare_ci( name_str, "seething_fury_deficit" ) )
+  {
+    return make_fn_expr( "seething_fury_deficit", [ this ] {
```

Comment:
Wonder if this is worth returning a constant zero if not running T30?

Reply:
I don't know if it's necessarily worth it but it seems like a sane thing to do so I've made the change